src/components/simulation/SimulationPanel.svelte

```diff
@@ -105,7 +105,8 @@
       { planId: $plan.id },
       null,
       user,
-      v => v[0]?.simulation_datasets,
+      //@ts-ignore
```

Comment:
What was being ignored here?

Reply:
It was complaining that `extent` had implicit type `any` - any tips on how to handle that?

Reply:
I added this type annotation:

```ts
({ extent, ...etc }: SimulationDataset & { extent: { extent: string } | null })
```